tests/src/api/global.rs

```diff
@@ -191,6 +193,25 @@ fn set_get_option() {
     assert!(!api::get_option::<bool>("modified").unwrap());
 }
 
+#[oxi::test]
+fn set_get_option_value() {
+    api::set_option_value(
+        "modified",
+        true.to_object().unwrap(),
+        &OptionValueOpts::builder()
+            .scope(nvim_oxi::api::opts::OptionScope::Global)
+            .build(),
+    )
+    .unwrap();
+    assert!(bool::from_object(api::get_option_value::<Object>(
+        "modified",
+        &OptionValueOpts::builder()
+            .scope(nvim_oxi::api::opts::OptionScope::Global)
+            .build(),
+    )
+    .unwrap()).unwrap());
+}
+
```

Comment:
we can refactor this to a) avoid calling `bool::{to,from}_object`, and b) build the `OptionValueOpts` once:

```rust
#[oxi::test]
fn set_get_option_value() {
    let opts = OptionValueOpts::builder()
        .scope(api::opts::OptionScope::Global)
        .build();

    api::set_option_value("modified", true, &opts).unwrap();

    assert!(api::get_option_value::<bool>("modified", &opts).unwrap());
}
```